src/util.cpp

```diff
@@ -8,18 +8,21 @@
  * @copyright Copyright (c) 2023
  * 
  */
+
 #include "util.hpp"
 
 #include "cli/conio.hpp"
 #include "consts.hpp"
 
 #include "Particle.h"
 
+/*! BYTES_PER_LINE
+    \brief A macro that stores the the size of each line of output as a constant 16 bytes
+*/
```

Comment:
Note that this still looks funny in the HTML documentation.  Please double check the entire HTML page - remember that all symbols show up at least twice, once in the brief, and once in the main body.

Reply:
The issue is the repeated "BYTES_PER_LINE" under Macro Definition Documentation, right? If I include \def in the docstring, that repeated line stops showing. Should I add that back in?

Reply:
Does it make sense to remove that line?

Reply:
I think so? Based on the way that other preexisting documentation looks. Sorry if I'm missing something.

Reply:
In that case, please remove the offending line

Reply:
Sounds good, just committed again